resources/assets/components/partials/profile/ProfileFollowers.vue

```diff
@@ -217,13 +217,13 @@
             },
 
             setCacheWarmTimeout() {
-                if(this.cacheWarmInterations >= 5) {
+                if(this.cacheWarmInteractions >= 5) {
                     this.isWarmingCache = false;
-                    swal('Oops', 'Its taking longer than expected to collect this account followers. Please try again later', 'error');
+                    swal('Oops', 'It is taking longer than expected to collect this account followers. Please try again later', 'error');
```

Comment:
`Its` is possessive, which is the wrong word. To avoid the confusion, I'm expanding the contraction (`It's`).